packages/filefeeds/src/filefeeds.ts

```diff
@@ -362,6 +362,7 @@ export class OneSchemaFileFeedsClass extends EventEmitter {
             /* local storage is not available, don't sweat it */
           }
         }
+        this.#hide()
```

Comment:
Thanks!

One question, though: should we just `#hide()` at this point, or `destroy()`?

Reply:
I think it should prob be destroy but lets have the change to destroy happen in a different pr along w/ the changes to close